### DISCUSSION TITLE
Distraction free editor (while typing)

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#3559] [Ideas]
When using the editor to work on text that requires a lot of attention, less visual impairment through UI interfaces can be aspired.

In other places this functionality is often called *distraction free* mode.

Eventually a toggle below "Full Width" could help fade out the UI elements in the top-left, top-right and bottom-right?

<details>

<summary>A little note on the 🖮</summary>

> Why is the keyboard hint sitting there for by default?
> Could its presence maybe be toggled by an independent switch, either in the profile (locally) or settings menu (globally)?

</details>

![Bildschirmfoto von 2022-05-19 21-25-17](https://user-images.githubusercontent.com/1645308/169387818-73545375-fb0e-48f7-b47f-f93c4303dd11.png)

**Comment by tommoor** [comment]
Work is already underway on this one here: https://github.com/outline/outline/pull/3503

**Reply by almereyda** [reply]
That went quick (:

**Comment by almereyda** [comment]
As a side note, this idea was heavily inspired by https://github.com/sethfair/logseq-focus-mode.
